src/constants/categories.js

```diff
@@ -254,20 +631,86 @@ export const CATEGORIES = [
     shortDescription: 'Aims to deliver coordinated and integrated policy, planning and investment across all levels of government.',
     subCategories: [
       {
+        name: 'Planning',
+        highlightColorLight: 'PLANNING_020',
+        highlightColorDark: 'PLANNING_500',
+        iconId: 'planningLocalGovernment',
+        summaryIndicatorIds: [
+          'indigenousPopulation',
+          'growthRate',
+          'population10yr',
+        ],
+        charts: [
+          {
+            name: 'Total population',
```

Comment:
Is this redundant? The chart does `const title = props.title || firstIndicator.name;` so for anything with one indicator there's no need to pass in a name.

Reply:
Best to be explicit. Indicators can be used in more than one chart.

Reply:
Oh, of course. 👍